projects/canopy/src/lib/forms/toggle/toggle.stories.common.ts

```diff
@@ -46,6 +52,10 @@ export class ReactiveToggleFormComponent {
     });
     this.form.valueChanges.subscribe((val) => this.toggleChange.emit(val));
   }
+
+  ngOnChanges(changes): any {
```

Comment:
do we need a return type here? can it be implicitly void

Reply:
The `any` was a mistake but I've added `void` instead as I think we should start to be more explicit with typings.